api/fasit.go

```diff
@@ -33,6 +34,7 @@ type FasitResource struct {
 	ResourceType string `json:"type"`
 	Properties   map[string]string
 	Secrets      map[string]map[string]string
+	Files        map[string]interface{}
```

Comment:
Should be called certificates, not files, if that is more precise

Reply:
Fixed.